pkgs/development/compilers/boo/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, pkgconfig, dbus, mono, makeWrapper, nant 
+, shared_mime_info, gtksourceview, gtk
+, targetVersion ? "4.5" }:
+
+let
+  release = "alpha";
+  version = "2013-10-21";
+in stdenv.mkDerivation {
```

Comment:
Usually, we use `rec` and define `version` within the attrset passed to `mkDerivation`